### ISSUE TITLE
Notepad++ installer runs at every upgrade

### ISSUE THREAD

**Issue by se35710** [lukesampson/scoop#1555]
auto-pr downgraded Notepad++ in commit [85d0d8d](https://github.com/lukesampson/scoop-extras/commit/85d0d8dd8e62fbce05ae056600bb405cbc2351f4).

Now at every `scoop upgrade *`, Notepad++ is reinstalled:

```
PS > scoop update *
[...]
Updating 'notepadplusplus' (7.4.1 -> 7.3.3)
Uninstalling 'notepadplusplus' (7.4.1)
Removing shim for 'notepad++'.
Unlinking C:\~\scoop\apps\notepadplusplus\current
Installing 'notepadplusplus' (7.3.3).
Loading npp.7.3.3.bin.x64.7z from cache
Checking hash of npp.7.3.3.bin.x64.7z... ok.
Extracting... done.
Linking C:\~\scoop\apps\notepadplusplus\current => C:\~\scoop\apps\notepadplusplus\7.3.3
Creating shim for 'notepad++'.
Creating shortcut for Notepad++ (notepad++.exe)
'notepadplusplus' (7.3.3) was installed successfully!
[...]
PS > scoop list
[...]
notepadplusplus (7.4.1)
[...]
```

**Comment by r15ch13** [comment]
Notepad++ version 7.4.0 and 7.4.1 have been withdrawn (see https://notepad-plus-plus.org/news/back-to-v7.3.3.html)
Reinstalling notepad++ will fix this or wait until they publish 7.4.1 again 😄

**Comment by se35710** [comment]
I am aware that those versions are withdrawn. But I guess it's related to #1072 and somewhat #1011, and with #1332 implemented it's difficult to fix it.
